### DISCUSSION TITLE
Cannot add issues to project from different organizations

### DISCUSSION THREAD

**Discussion by LoneDev6** [community/community#54338] [Projects and Issues]
### Select Topic Area

Bug

### Select Feature Area

- [ ] Issues
- [X] Projects

### Body

The UI doesn't allow to add issues from other organizations, [this workflow action](https://github.com/actions/add-to-project) does allow that, probably because it's using the Github API. 

I'd love to be able to add issues from other organizations to my project without using the custom workflow action.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by ssbarnea** [comment]
I can confirm that while this was declared as working it seems not to be the case. In some case I was able to get the auto completer to suggest issues from other organization via `#orgname/project` but to my surprise it did only suggest already closed issues and PRs on that project, so I was not able to add an open one.

In one case I was able to select one issue but the assignment was only visible inside the project board. On the issue itself it kept reporting as not being assigned to any project.

![](https://sbarnea.com/ss/Screen-Shot-2023-05-19-06-59-31.58.png)

**Comment by LoneDev6** [comment]
Github action used: https://github.com/actions/add-to-project

**Comment by sakyork** [comment]
Friendly bumping this thread. We're currently piloting Issues as a potential Jira replacement, and it would be great to add the projects directly on the Issue without taking the extra step of doing so on the board.

**Comment by cratelyn** [comment]
bumping this issue myself.
